src/components/NoMatch/NoMatch.tsx

```diff
@@ -0,0 +1,19 @@
+import { Link, useLocation } from 'react-router-dom'
+import './NoMatch.css'
+
+const NoMatch = (): JSX.Element => {
+  const location = useLocation();
+  console.log(location, "is this running")
+  return (
+    <div>
+      <h3 className="cy-error-message">
+        No match for <code>{location.pathname}</code>
```

Comment:
This is a good, descriptive message for our user. What do you think about adding a button for the user to return to the feed?